src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -383,7 +383,8 @@ export function GetCorpInfoObject (cd: CorpTypeCd): CorpInfoIF {
  * @returns the description (or '' if not found)
  */
 export function GetCorpFullDescription (cd: CorpTypeCd): string {
-  return CorpInfoArray.find(obj => (cd === obj.corpTypeCd))?.fullDesc || ''
+  const item = CorpInfoArray.find(obj => (cd === obj.corpTypeCd))
+  return (item && item.fullDesc) || ''
```

Comment:
Strangely, the optional chaining worked previously, but not once this module was imported into enums. Lots of Storybook errors. 👻